packages/fxa-settings/src/components/WarningMessage/index.tsx

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+type WarningMessageProps = {
+  children: string;
+  warningMessageFtlId: string;
+  warningType: string;
+};
+
+const WarningMessage = ({
+  children,
+  warningMessageFtlId,
+  warningType,
+}: WarningMessageProps) => {
+  return (
+    <div className="my-8 px-4 text-sm" data-testid="warning-message-container">
```

Comment:
I don't think we need `px-4`. When we'd done the Tailwindification stuff for some of content-server, Design said it was fine to go full paragraph width. Also I think `my-8` should be `mt-5 mb-8`, in prod I'm seeing 20px total space between the "to continue" text and the "note" text.

Btw, I think you're missing the semicolon in `NOTE:` 😬